src/resolvers.ts

```diff
@@ -30,21 +32,37 @@ export const resolvers = {
       await AppDataSource.initialize();
       console.log('Conectado ao db local!');
 
-      const user = new User();
-      user.name = args.data.name;
-      user.email = args.data.email;
-      user.password = args.data.password;
-      user.birthDate = args.data.birthDate;
-
-      await AppDataSource.manager.save(user);
-      await AppDataSource.destroy();
-
-      return {
-        id: user.id,
-        name: user.name,
-        email: user.email,
-        birthDate: user.birthDate,
-      };
+      const emailsList = await AppDataSource.getRepository(User).find({
```

Comment:
```suggestion
      const emails = await AppDataSource.getRepository(User).find({
```
O que acha de ser `emails`? O sufixo List nesse caso é uma redundancia, já que sem ele dá pra entender que o retorno é um array de emails.